src/Simplex/Messaging/Crypto.hs

```diff
@@ -98,16 +97,18 @@ data CryptoError
   | CryptoCipherError CE.CryptoError
   | CryptoIVError
   | CryptoDecryptError
+  | CryptoLargeMsgError
+  | CryptoHeaderError String
   deriving (Eq, Show, Exception)
 
 pubExpRange :: Integer
 pubExpRange = 2 ^ (1024 :: Int)
 
-aeKeySize :: Int
```

Comment:
`ae` stood for "authenticated encryption" here, `aes` is a cypher name and tag is separate from a cypher (it is part of authenticated encryption but implemented not using AES itself, no?). Maybe use different prefix at least for `tagSize`?

Reply:
maybe - not sure. I was confused with ae prefix.